Undefined names: TOTAL_BAR_LENGTH, last_time, term_width

Perhaps these four lines should not be commented out.  https://github.com/kuangliu/pytorch-retinanet/blob/master/utils.py#L201-L204

flake8 testing of https://github.com/kuangliu/pytorch-retinanet on Python 2.7.13

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./utils.py:211:19: F821 undefined name 'TOTAL_BAR_LENGTH'
    cur_len = int(TOTAL_BAR_LENGTH*current/total)
                  ^

./utils.py:212:20: F821 undefined name 'TOTAL_BAR_LENGTH'
    rest_len = int(TOTAL_BAR_LENGTH - cur_len) - 1
                   ^

./utils.py:223:28: F821 undefined name 'last_time'
    step_time = cur_time - last_time
                           ^

./utils.py:235:20: F821 undefined name 'term_width'
    for i in range(term_width-int(TOTAL_BAR_LENGTH)-len(msg)-3):
                   ^

./utils.py:235:35: F821 undefined name 'TOTAL_BAR_LENGTH'
    for i in range(term_width-int(TOTAL_BAR_LENGTH)-len(msg)-3):
                                  ^

./utils.py:239:20: F821 undefined name 'term_width'
    for i in range(term_width-int(TOTAL_BAR_LENGTH/2)):
                   ^

./utils.py:239:35: F821 undefined name 'TOTAL_BAR_LENGTH'
    for i in range(term_width-int(TOTAL_BAR_LENGTH/2)):
                                  ^

7       F821 undefined name 'TOTAL_BAR_LENGTH'
```

--- a/utils.py
+++ b/utils.py
@@ -198,11 +198,11 @@ def msr_init(net):
         elif type(layer) == nn.Linear:
             layer.bias.data.zero_()
 
-# _, term_width = os.popen('stty size', 'r').read().split()
-# term_width = int(term_width)
-# TOTAL_BAR_LENGTH = 86.
-# last_time = time.time()
-# begin_time = last_time
+_, term_width = os.popen('stty size', 'r').read().split()
+term_width = int(term_width)
+TOTAL_BAR_LENGTH = 86.
+last_time = time.time()
+begin_time = last_time
 def progress_bar(current, total, msg=None):
     global last_time, begin_time
     if current == 0:

